devel/ansible/roles/dev/files/sar.py

```diff
@@ -0,0 +1,119 @@
+import click
+import requests
```

Comment:
Any reason in particular for not using `python-freeipa` like everything else?

Reply:
Nothing major. We haven't swapped the client yet to the new version and we spoke about maybe letting our intern James take that task on. I saw no real drawback to using `requests` since it is essentially just two methods. I was thinking it would be good for James to start with this file when he migrates the client to the new version.